Userland/Utilities/ls.cpp

```diff
@@ -484,7 +484,8 @@ static bool print_names(const char* path, size_t longest_name, const Vector<Stri
         printed_on_row += column_width;
 
         for (size_t j = nprinted; i != (names.size() - 1) && j < column_width; ++j)
-            printf(" ");
+            if (isatty(STDOUT_FILENO))
```

Comment:
Please cache the result of `isatty(STDOUT_FILENO)` somewhere instead of calling it over and over.
Also please wrap multi-line `for` blocks in `{` and `}`.